src/test/javascript/spec/service/text-exercise.service.spec.ts

```diff
@@ -10,12 +10,20 @@ import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
 import { Router } from '@angular/router';
 import { MockSyncStorage } from '../helpers/mocks/service/mock-sync-storage.service';
 import { MockRouter } from '../helpers/mocks/mock-router';
+import { HttpResponse } from '@angular/common/http';
+import * as chai from 'chai';
+import * as moment from 'moment';
+import { StudentParticipation } from 'app/entities/participation/student-participation.model';
+
+const expect = chai.expect;
 
 describe('TextExercise Service', () => {
     let injector: TestBed;
     let service: TextExerciseService;
     let httpMock: HttpTestingController;
     let elemDefault: TextExercise;
+    let requestResult: any;
```

Comment:
```suggestion
    let requestResult: HttpResponse<TextExercise>;
```

Reply:
This change actually breaks my tests because i assign an EntityArrayResponseType to this in line 93.